pkg/common/configs/configvalidator.go

```diff
@@ -573,6 +573,19 @@ func checkLimit(limit Limit, existedUserName map[string]bool, existedGroupName m
 	return nil
 }
 
+func checkLimitsStructure(rootQueue *QueueConfig, partitionLimits []Limit) error {
+	if len(partitionLimits) > 0 && len(rootQueue.Limits) > 0 {
```

Comment:
this change is a kind of behavior change, so it deserves a docs. Could you please file a PR as follow-up to reminder user?

Reply:
I'll raised another pr after this pull-request merged. Thanks !